website/source/docs/internals/providers/new-provider.html.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: "docs"
+page_title: "Creating Providers"
+sidebar_current: "docs-internals-provider-guide-new-provider"
+description: |-
+  How to get started building a new provider.
+---
+
+# Creating Providers
+
+Say you've found a new API provider that Terraform doesn't support, and you
```

Comment:
Yeah, I mentioned I had only updated the first few pages. Sorry for the confusion, will try to be clearer next time. Was trying to save us both some time and spot-check for the voice you wanted before I rewrote a 7000 word document (or made you re-edit a 7000 word document).